### ISSUE TITLE
Bug in velocity

### ISSUE THREAD

**Issue by rocketnik** [OfTheWolf/UBottomSheet#63]
Please take a look at
`if velocity.y < 100 { /// dragging up`
It should be
`if velocity.y < -100 { /// dragging up`

As I understand this is for fast dragging which should always select the next point in direction of the drag instead of find the closes point, which could mean going back a bit.

Having the sheet almost closed (dragged to bottom) and then moving slightly (not matter if up or down) of speed of 5 or -5, it will be below 100 and interpreted as fast drag up, which it is not. It would be a drag up, if velocity was negative and it would be a fast drag up, if velocity was below -100.

**Comment by OfTheWolf** [comment]
@rocketnik That's correct. Also /// dragging up, down comments are misleading and should be removed.

**Comment by rocketnik** [comment]
To my understanding the comments are correct. Maybe add the word “fast” before “drag”
